Message.php

```diff
@@ -291,6 +291,24 @@ public function addTo($email, $name = '', $type = 'to')
     }
 
     /**
+     * Replace all recipients by a specific recipient
+     *
+     * @param string $email
+     * @param string $name
+     * @param string $type
+     *
+     * @return Message
+     */
+    public function replaceTo($email, $name = '', $type = 'to')
+    {
+        $this->to = array(
+            array('email' => $email, 'name' => $name, 'type' => $type)
+        );
```

Comment:
It's better to reset the ``$to`` property and use ``addTo`` method inside your new ``replaceTo`` method.